pkg/kubelet/kubelet_node_status.go

```diff
@@ -158,6 +159,38 @@ func (kl *Kubelet) tryRegisterWithApiServer(node *v1.Node) bool {
 	return false
 }
 
+//updateDefaultLabels will set the default labels on the node
+func (kl *Kubelet) updateDefaultLabels(initialNode, existingNode *v1.Node) bool {
+	defaultLabels := []string{
+		kubeletapis.LabelHostname,
+		kubeletapis.LabelZoneFailureDomain,
+		kubeletapis.LabelZoneRegion,
+		kubeletapis.LabelInstanceType,
+		kubeletapis.LabelOS,
+		kubeletapis.LabelArch,
+	}
+
+	if existingNode.Labels == nil {
```

Comment:
Turns out this only happened during unit tests where other tests had node objects without labels.
I updated the tests and removed this block in 5a1587f9f9fe622c2b2cb1a762d18d4b04c5bb76